crds/README.md

```diff
@@ -0,0 +1,30 @@
+## CRD's Schema<br/>
```

Comment:
remove - this is internal. I thought we had mitigation so this file does not show up here, will need to check why it did not work.